docs/faststore/docs/customization/building-sections/override-use-cases/adding-more-icon-options-to-the-alert-component.mdx

```diff
@@ -0,0 +1,126 @@
+
+        ```tsx focus=3:7
+        import { AlertSection, getOverriddenSection } from '@faststore/core'
+
+        const CustomIconsAlert = getOverriddenSection({
+        Section: AlertSection,
+        })
+
+        export default CustomIconsAlert
+        ```
+
+    - This is an Alert component override with custom icon options in the Headless CMS. The component is defined using `getOverriddenSection`.
+    - This code changes only the schema to include more options for native Icons supported by `@faststore/ui`.
+    - The override options specify the `Section` to be overridden (Alert).
+
+    > ℹ️ For further details on code implementation, see the [`CustomIconsAlert`](https://github.com/vtex-sites/playground.store/tree/main/src/components/sections/CustomIconsAlert) folder available in the [playground.store](https://github.com/vtex-sites/playground.store) repository.
+
+### Synchronizing the changes with the Headless CMS
+
+Add the section to the Headless CMS by following the instructions available in [syncing components to the Headless CMS](https://developers.vtex.com/docs/guides/faststore/overrides-syncing-components-with-the-headless-cms).
+
+The following schema was used as an example in [`cms/faststore/sections.json`](https://github.com/vtex-sites/playground.store/blob/main/cms/faststore/sections.json#L2-L57):
+
+```json
+[
+  {
+    "name": "CustomIconsAlert",
+    "schema": {
+      "title": "Alert with more Icon options",
+      "description": "Add an alert that has more Icon options",
+      "type": "object",
+      "required": ["icon", "content", "dismissible"],
+      "properties": {
+        "icon": {
+          "type": "string",
+          "title": "Icon",
+          "enumNames": [
+            "Bell",
+            "BellRinging",
+            "Checked",
+            "Info",
+            "Truck",
+            "User",
+            "Plus",
+            "PlusCircle",
+            "Ruler",
+            "Star",
+            "Storefront",
+            "Twitter"
+          ],
+          "enum": [
+            "Bell",
+            "BellRinging",
+            "Checked",
+            "Info",
+            "Truck",
+            "User",
+            "Plus",
+            "PlusCircle",
+            "Ruler",
+            "Star",
+            "Storefront",
+            "Twitter"
+          ]
+        }
+      }
+    }
+  }
+]
+```
+
+> ℹ️ For further details on code implementation, see the [`sections.json`](https://github.com/vtex-sites/playground.store/blob/main/cms/faststore/sections.json#L1-L55) file available in the [playground.store](https://github.com/vtex-sites/playground.store) repository.
+
+## Results
+
+Once you have [set your development mode](https://developers.vtex.com/docs/guides/faststore/headless-cms-previewing-changes-in-development-mode) to see the changes locally, you can see the new Alert icon. Below, you can check an example storefront before and after the instructions in this guide:
```

Comment:
Is it clear that you will see the changes in the Alert section (dropdown icons with more icons)? You'll have to choose the new icon and save the changes before you can see the new icon applied.